### PR TITLE
Add package source mapping to NuGet.Jobs

### PR DESCRIPTION
**Pull request by shishirx34**
While working on CPM needed to add package source mapping. Probably good idea to have it irrespectively as a best practice. Ignore other commits, I will be squashing while merging.

### CODE REVIEW

**Review by joelverhagen — DISMISSED** [review]

File: .nuget/NuGet.config [hunk 1]
```diff
@@ -8,6 +8,122 @@
...
+      <package pattern="nerdbank.streams" />
+      <package pattern="netstandard.library" />
+      <package pattern="newtonsoft.json" />
+      <package pattern="nuget.build.tasks.pack" />
+      <package pattern="nuget.common" />
+      <package pattern="nuget.configuration" />
+      <package pattern="nuget.frameworks" />
+      <package pattern="nuget.packaging" />
+      <package pattern="nuget.protocol" />
+      <package pattern="nuget.versioning" />
```
Comment (zhhyu) [inline .nuget/NuGet.config:91]: It seems that these packages have been mapped in "nuget-build" feed, is it expected to map here again?

Comment (shishirx34) [reply .nuget/NuGet.config:91]: You are right, I didn't notice it. This mapping was generated via `packagesourcemapper` tool. Perhaps it picked up packages restored from multiple feeds (where available). I'll remove this and keep it in `nuget-build` feed.

Comment (shishirx34) [reply .nuget/NuGet.config:91]: ok. I checked with Erick. It looks like these mappings are correct. The mapping tool doesn't provide different version level granularity and this mapping will be needed for proper build.

**Review by zhhyu — DISMISSED** [review]

File: .nuget/NuGet.config (hunk 1 above)
Comment (shishirx34) [reply .nuget/NuGet.config:55]: These are indeed in nuget.org feed.

**Review by joelverhagen — APPROVED** [review]

**Pull request merged**